src/navbar/navbar.component.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+angular.module('navbar').component('navbar', {
+  templateUrl: 'navbar/navbar.template.html',
+  bindings: {},
+  controller: ['Auth', '$state', function BooksListController(Auth, $state) {
```

Comment:
use arrow

Reply:
Is a component, need to have a new context.